api/types/plugins/logdriver/entry.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
Agreed, can we do this in another PR? I would prefer not to refactor everything here.